docs/release-notes/release-notes-0.18.5.md

```diff
@@ -50,6 +50,15 @@
 
 * [Improved user experience](https://github.com/lightningnetwork/lnd/pull/9454)
  by returning a custom error code when HTLC carries incorrect custom records.
+
+* [Make input validation stricter](https://github.com/lightningnetwork/lnd/pull/9470)
+  when using the `BumpFee`, `BumpCloseFee` and `BumpForceCloseFee` RPCs. For the
+  `BumpFee` RPC the new param deadline-delta is introduced. For the other two
+  RPCs the conf_target param is added to the `lncli`. The conf_target is now
```

Comment:
You mean `deadline_delta` is added since `conf_target` already exists?

Reply:
hmm rephrased it, but for the bumpforceclose RPC, I added it now because the lncli had it but it was interpreted as deadline, and now the RPC level has to keep these commands in sync.